app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class='mod-header'>
+  <div class='layout-wrap'>
+    <h1><a href="http://turing.io">Turing School of Software & Design</a></h1>
+    <nav>
+      <ul>
+        <li><a href="/">Today</a></li>
+        <li><a href="/all/">All Outlines</a></li>
+        <li><a href="http://turing.io/">Main Site</a></li>
+      </ul>
+    </nav>
+  </div>
+</header>
```

Comment:
We don't want it to say today, all outlines, main site on our page. but this at least gives us the nav bar frame work to then update and style from there.